app/experimenter/targeting/constants.py

```diff
@@ -461,6 +461,20 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+NEW_USERS_WITH_INFREQUENT_USE = NimbusTargetingConfig(
+    name="New users with infrequent use",
+    slug="new_users_with_infrequent_use",
+    description="0 - 6 days activity in past 28 days and profile age <= 28 days",
+    targeting=(
+        f"{PROFILE28DAYS} "
+        "&& userMonthlyActivity|length <= 6"
+    ),
```

Comment:
Oh I got this formatting error in my previous PR. Try this?
```suggestion
    targeting=f"{PROFILE28DAYS} && userMonthlyActivity|length <= 6",
```

Reply:
I can fix it don't worry about it